collectors/build/sonar/src/main/java/com/capitalone/dashboard/collector/SonarCollectorTask.java

```diff
@@ -211,4 +272,17 @@ private boolean isNewQualityData(SonarProject project, CodeQuality codeQuality)
         return codeQualityRepository.findByCollectorItemIdAndTimestamp(
                 project.getId(), codeQuality.getTimestamp()) == null;
     }
+    
+    private long convertToTimestamp(String date) {
+    	SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssZ");
+    	Date d;
+    	try {
+    		d = sdf.parse(date);
+    		return d.getTime();
+    	} catch (ParseException e) {
+    		LOG.error(e);
+    		return 0;
+    	}		
+    }
+
```

Comment:
Use joda-time and then use simple: new DateTime(dateTime).getMillis();

Reply:
Is this specifically for the timestamp conversion?